static/styles/zulip.css

```diff
@@ -1711,9 +1713,13 @@ div.focused_table {
     background-color: transparent;
 }
 
-.next_is_same_sender {
-    border-bottom: 0;
-    padding-bottom: 0;
+/* TODO: this didn't seem to do anything anymore,
+   and now that we're using padding again, we don't
+   ever want to remove the bottom padding completely.
+ */
+.beside_same_sender {
```

Comment:
cc @amanagr -- also would love your thoughts on this whole PR! thank you :)

Reply:
I guess you can wait for #24070 to get merged.